packages/graph/hash_graph/lib/graph/src/ontology/data_type.rs

```diff
@@ -11,24 +11,149 @@ use crate::store::query::{OntologyPath, ParameterType, QueryRecord, RecordPath};
 /// A path to a [`DataType`] field.
 ///
 /// Note: [`DataType`]s currently don't reference other [`DataType`]s, so the path can only be a
-/// single field.
+/// single field. This means, that the path currently will always be a sequence with only one
+/// element.
 ///
 /// [`DataType`]: type_system::DataType
 // TODO: Adjust enum and docs when adding non-primitive data types
 //   see https://app.asana.com/0/1200211978612931/1202464168422955/f
 #[derive(Debug, PartialEq, Eq)]
 pub enum DataTypeQueryPath {
-    VersionId,
+    /// The [`BaseUri`] of a [`DataType`].
```

Comment:
I noticed that for entity we used `this` and here we use both `a` and `the` to talk about the kind. We should be consistent across the `*QueryPath` docs :)